srcCpp/RTIDDSImpl.cxx

```diff
@@ -2378,18 +2362,18 @@ IMessagingWriter *RTIDDSImpl<T>::CreateWriter(const char *topic_name)
         return NULL;
     }
 
-    if (strcmp(topic_name, perftest_cpp::_ThroughputTopicName) == 0) {
+    if (strcmp(topic_name, THROUGHPUT_TOPIC_NAME) == 0) {
         qos_profile = "ThroughputQos";
-    } else if (strcmp(topic_name, perftest_cpp::_LatencyTopicName) == 0) {
+    } else if (strcmp(topic_name, LATENCY_TOPIC_NAME) == 0) {
         qos_profile = "LatencyQos";
-    } else if (strcmp(topic_name, perftest_cpp::_AnnouncementTopicName) == 0) {
+    } else if (strcmp(topic_name, ANNOUNCEMENT_TOPIC_NAME) == 0) {
         qos_profile = "AnnouncementQos";
     } else {
         fprintf(stderr,
                 "topic name must either be %s or %s or %s.\n",
-                perftest_cpp::_ThroughputTopicName,
-                perftest_cpp::_LatencyTopicName,
-                perftest_cpp::_AnnouncementTopicName);
+                THROUGHPUT_TOPIC_NAME,
+                LATENCY_TOPIC_NAME,
+                ANNOUNCEMENT_TOPIC_NAME);
```

Comment:
Duplicate code (line 2655). May be new function with that?

Reply:
@alfonso-soto 
Good idea. Could you create a task for that so we don't forget? For the next release.